### ISSUE TITLE
IllegalStateException: Already connected caused by logger at level DEBUG 

### ISSUE THREAD

**Issue by avelinsk** [salesforce-marketingcloud/FuelSDK-Java#66]
```java
ETRestConnection.java  
  if (logger.isDebugEnabled()) {
            for (String key : connection.getRequestProperties().keySet()) {
                logger.debug(key + ": " + connection.getRequestProperty(key));
            }
        }
```

If the logger is at INFO, no RTE is thrown.

**Comment by avelinsk** [comment]
Affects release 1.2.1

**Comment by avelinsk** [comment]
```
Caused by: java.lang.IllegalStateException: Already connected
        at sun.net.www.protocol.http.HttpURLConnection.getRequestProperties(HttpURLConnection.java:3142)
        at sun.net.www.protocol.https.HttpsURLConnectionImpl.getRequestProperties(HttpsURLConnectionImpl.java:358)
        at com.exacttarget.fuelsdk.ETRestConnection.sendRequest(ETRestConnection.java:278)
        at com.exacttarget.fuelsdk.ETRestConnection.sendRequest(ETRestConnection.java:223)
        at com.exacttarget.fuelsdk.ETRestConnection.post(ETRestConnection.java:135)
        at com.exacttarget.fuelsdk.ETClient.requestToken(ETClient.java:330)
        at com.exacttarget.fuelsdk.ETClient.requestToken(ETClient.java:292)
        at com.exacttarget.fuelsdk.ETClient.<init>(ETClient.java:156)
        at com.exacttarget.fuelsdk.ETClient.<init>(ETClient.java:117)
        at com.coremedia.blueprint.marketing.connector.SFMarketingCloudClient.afterPropertiesSet(SFMarketingCloudClient.java:32)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor$LifecycleElement.invoke(InitDestroyAnnotationBeanPostProcessor.java:366)
        at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor$LifecycleMetadata.invokeInitMethods(InitDestroyAnnotationBeanPostProcessor.java:311)
        at org.springframework.beans.factory.annotation.InitDestroyAnnotationBeanPostProcessor.postProcessBeforeInitialization(InitDestroyAnnotationBeanPostProcessor.java:134)
        ... 45 common frames omitted
```

**Comment by smunuswami** [comment]
We are looking in to this issue. We will get back to you.

**Comment by avelinsk** [comment]
great thanks

**Comment by sharif26** [comment]
That issue is resolved on v1.2.2, please use that and let us know if we face any difficulty.
